### DISCUSSION TITLE
automatic deletion of favorites?

### DISCUSSION THREAD

**Discussion by unusualevent** [mastodon/mastodon#29161] [Q&A]
Does the post auto-deletion feature also include deleting favorites?

How would you go about unfavoriting everything?

**Comment by mjankowski** [comment]
Within the auto delete prefs page, there's an option to preserve favorites ... is that what you are looking for?

**Reply by evanchaney** [reply]
I suspect the questions are:

1. If I have automatic post deletion turned on will it also automatically un-favorite posts I favorited? (The answer appears to be no it will not.)
2. If not, how can I bulk un-favorite posts I favorited?

For example, if you are trying to clean up your digital footprint and have favorited a lot of posts over a lot of years.

**Comment by zesty** [comment]
are boosts/re-posts included under the notion of posts to be deleted?

**Reply by evanchaney** [reply]
Yes. Boosts are included.

**Comment by zesty** [comment]
because some favorites are used to prevent deletion of one's own posts, there would need to be an option to not delete favorites of one's own posts

**Comment by ClearlyClaire** [comment]
The post auto-deletion feature deletes boosts, replies and standalone posts, but not bookmarks or favorites. There is currently no built-in feature to automatically delete those.

**Comment by GeraldSoellinger** [comment]
Does somebody know a method/tool (e.g. third party client) which can also auto delete favorites when certain conditions are met?
